controllers/glance_controller.go

```diff
@@ -699,6 +687,10 @@ func (r *GlanceReconciler) apiDeploymentCreateOrUpdate(ctx context.Context, inst
 		apiSpec.GlanceAPITemplate.NodeSelector = instance.Spec.NodeSelector
 	}
 
+	// Inherit the values required for PVC creation from the top-level CR
+	apiSpec.GlanceAPITemplate.StorageRequest = instance.Spec.StorageRequest
```

Comment:
All the instances needs to be tied to the same `StorageClass` and the same `StorageRequest`, hence this propagates the two fields to the `statefulset` that is supposed to create them via `VolumeClaimTemplates`